demos/serverless/package.json

```diff
@@ -8,7 +8,8 @@
   "dependencies": {
     "aws-sdk": "^2.664.0",
     "fs-extra": "^9.0.0",
-    "uuid": "^8.2.0"
+    "uuid": "^8.2.0",
+    "aws-embedded-metrics": "latest"
```

Comment:
Recommend specifying the version with the `^` prefix (`^2.0.2`) because a new major version might break the serverless demo.

Reply:
Fixed